schema/deploy/tables/naics.sql

```diff
@@ -0,0 +1,42 @@
+-- Deploy ggircs-portal:tables/naics to pg
+-- requires: schema_ggircs_portal
+
+begin;
+
+create table ggircs_portal.naics (
+  naics_code integer not null primary key,
```

Comment:
same comment regarding naics_code being a varchar. Feel free to add a generated integer id and a unique index.